sparkpost/base.py

```diff
@@ -33,7 +33,7 @@ def uri(self):
 
     def request(self, method, uri, **kwargs):
         headers = {
-            'User-Agent': 'python-sparkpost/' + sparkpost.__version__,
+            'User-Agent': 'python_sparkpost/' + sparkpost.__version__,
```

Comment:
I think we need to change the name of the module to have an underscore instead of a dash in order to be able to be imported into the web module.